src/file-io.c

```diff
@@ -64,245 +64,15 @@ int save_count = 0;
  * upgrading from old versions of GTT.  All 'saves' are in the new
  * file format (currently, GConf-2).
  *
- * 1) Oldest format is data stuck into a ~/.gtimetrackerrc file
- *    and is handled by the project_list_load_old() routine.
- * 2) Next is Gnome-1 Gnome-Config files in ~/.gnome/gtt
- * 3) Next is Gnome-2 Gnome-Config files in ~/.gnome2/GnoTime
- * 4) Current is GConf2 system.
+ * 1) Next is Gnome-1 Gnome-Config files in ~/.gnome/gtt
+ * 2) Next is Gnome-2 Gnome-Config files in ~/.gnome2/GnoTime
+ * 3) Current is GConf2 system.
```

Comment:
This first confused me a little because I read it as the priority order in which it attempts to read. Which is wrong of course, it's the chronological order in which the various methods were added to the code - and they are tried in the reverse order relative to this list. One might add a brief sentence to make that clear.

As a minor matter, it's a little strange when point 1 begins with "Next", and in the untouched comment a few lines above this, "that" is misspelled.

Reply:
I was thinking something like this:
```c
/* ============================================================= */
/* Configuration file I/O routines:
 * Note that this file supports reading from several old, 'obsolete'
 * config file formats that GTT has used over the years.  We support
 * these reads so that users do not get left out in the cold when
 * upgrading from old versions of GTT.  All 'saves' are in the new
 * file format (currently, GConf-2/GSettings).
 *
 * 1) First initialize GSettings.
 * 2) Then initialize GConf-2.
 * 3) Otherwise, try Gnome-2 Gnome-Config files in ~/.gnome2/GnoTime
 * 4) Otherwise, try Gnome-1 Gnome-Config files in ~/.gnome/gtt
 * 
```

Reply:
I reworked the comment similar to your suggestion.